src/ui/pages/IncomingRequest/components/BallotTransactionRequest.tsx

```diff
@@ -0,0 +1,140 @@
+import { IonText } from "@ionic/react";
+import { useMemo } from "react";
+import { i18n } from "../../../../i18n";
+import CardanoLogo from "../../../assets/images/CardanoLogo.jpg";
+import {
+  CardDetailsBlock,
+  CardDetailsItem,
+} from "../../../components/CardDetails";
+import { PageFooter } from "../../../components/PageFooter";
+import { PageHeader } from "../../../components/PageHeader";
+import { ScrollablePageLayout } from "../../../components/layout/ScrollablePageLayout";
+import { RequestProps } from "../IncomingRequest.types";
+import "./BallotTransactionRequest.scss";
+
+const BallotTransactionRequest = ({
```

Comment:
The name of the component is "Sign transaction". "Ballot" is just one of the things you can sign. You should rename every instance of "ballot" to "sign transaction".

![14052024092131](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/6a3a3718-3414-41e1-a651-03a6416d7947)